extensions/ql-vscode/src/databases/database-fetcher.ts

```diff
@@ -590,7 +590,7 @@ export async function convertGithubNwoToDatabaseUrl(
       repo,
     });
 
-    const languages = response.data.map((db: any) => db.language);
+    const languages = response.data.map((db) => db.language);
```

Comment:
In this case the correct type is inferred, so we're not just getting an implicit `any` type here.